src/Arcus.Security.Core/Storage/Source/EnvironmentVariableSecretProvider.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
```

Comment:
Let's move this to `Arcus.Security.Core/Providers`

Reply:
Done

Reply:
Done